README.md

```diff
@@ -53,6 +53,38 @@ You can first localize the robot by initializing the pose and then setting a nav
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demos:
+
+### Host/container insider attacker
+``` bash
+rocker rosswg/turtlebot3_demo "byobu -f configs/reconnaissance_demo/unsecure.conf attach"
+```
+
+### Internal network attacker
+``` bash
+# in Terminal (terminal 1), initialize first a swarm
+docker swarm init
+# in Terminal (terminal 1), create the network overlay
+docker network create -d overlay \
+  --subnet=10.0.0.0/24 \
+  --gateway=10.0.0.1 \
+	--ip-range 10.0.0.192/27 \
+  --attachable \
+  overlay
```

Comment:
Isn't `overlay` already a commonly/default network name used docker?
Might want to try if to name it something else, or see if the default `overlay` is enough.

Reply:
I originally wanted to name it ros2net (and avoid potential confusions) but I then realized that rocker is hardcoding the networking options making it a bit difficult to join other-than-predefined networks. Tried a few approaches (e.g. using https://github.com/sloretz/off-your-rocker#docker-run-args) but eventually decided to avoid submitting PRs to rocker (didn't wanted to risk not getting it soon enough or having part of the audience with previous versions).

`overlay` is indeed a networking driver type but no network named `overlay` by default shows up in my docker instalation (tested in OS X):

```bash
victor:~/turtlebot3_demo (master)$
| => docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
9ca6f66f5c4a        bridge              bridge              local
debdd4d51bf3        docker_gwbridge     bridge              local
d6b4095558bd        host                host                local
k7fwav1rlz1o        ingress             overlay             swarm
ad76f78c7ba0        none                null                local
```

Can you confirm this last bit please?

Reply:
Ahh, newer installs on ubuntu seem to have changes to only create a default bridge instead:

```
$ docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
16fa8b27fe17        bridge              bridge              local
52ba16a29500        host                host                local
e688be1ac788        none                null                local
```

> rocker is hardcoding the networking options

Hmm, that's annoying. 

https://github.com/osrf/rocker/blob/8f0da1f49c7240efc96ffa4ffa48101af580911e/src/rocker/cli.py#L37

Ok, let's stick with what you have working for now.

Reply:
In any case, I've opened a ticket about this here: https://github.com/osrf/rocker/issues/70

Reply:
Let me resolve this for now @ruffsl.